etc/profile-a-l/fractal.profile

```diff
@@ -0,0 +1,49 @@
+# Firejail profile for fractal
+# Description: Desktop client for Matrix 
+# This file is overwritten after every install/update
+# Persistent local customizations
+include fractal.local
+# Persistent global definitions
+include globals.local
+
+noblacklist ${HOME}/.cache/fractal
+
+include disable-common.inc
+include disable-devel.inc
+include disable-exec.inc
+include disable-interpreters.inc
+include disable-passwdmgr.inc
+include disable-programs.inc
+include disable-shell.inc
+include disable-xdg.inc
+
+mkdir ${HOME}/.cache/fractal
+whitelist ${HOME}/.cache/fractal
+whitelist ${DOWNLOADS}
+include whitelist-common.inc
+include whitelist-runuser-common.inc 
+include whitelist-usr-share-common.inc
+include whitelist-var-common.inc
+
+apparmor
+caps.drop all
+netfilter
+nodvd
+nogroups
+nonewprivs
+noroot
+notv
+nou2f
+protocol unix,inet,inet6
+seccomp
+shell none
+tracelog
+
+disable-mnt
+private-bin fractal
+private-cache
+private-dev
+private-etc alsa,alternatives,asound.conf,ca-certificates,crypto-policies,fonts,gtk-2.0,gtk-3.0,host.conf,hostname,hosts,ld.so.cache,ld.so.conf,ld.so.conf.d,ld.so.preload,locale,locale.alias,locale.conf,mime.types,nsswitch.conf,pki,pulse,resolv.conf,selinux,ssl,X11,xdg
+private-tmp
+
+dbus-system none
```

Comment:
From flatpak
```
dbus-user filter
dbus-user.own org.gnome.Fractal
dbus-user.talk ca.desrt.dconf
dbus-user.talk org.freedesktop.secrets
dbus-user.talk org.freedesktop.Notifications
```

Reply:
I had two more using dbus-monitor something secret identity etc. Allowing this many vs. allowing all -which is better?

Reply:
A lax filter rule is still bedder then no rules at all.

Reply:
I have:
```
ca.desrt.dconf.Writer
org.freedesktop.Secret.Service
org.freedesktop.Secret.Item
org.freedesktop.Secret.Collection
```

Add all?

Reply:
talk/own rules are for _names_ (e.g. ca.desrt.dconf), ca.desrt.dconf.Writer is a _interface_.